package.json

```diff
@@ -222,7 +222,7 @@
         ]
       },
       {
-        "permissionName": "ui-eholdings.package-title.select-unselect",
+        "permissionName": "ui-eholdings.package-title.execute",
```

Comment:
I think we should still have `select-unselect` because without it it's impossible to tell what we're executing
So, maybe call it `ui-eholdings.package-title.select-unselect.execute`